Services/UI/OJS.Services.Ui.Data/Implementations/SubmissionsForProcessingDataService.cs

```diff
@@ -86,10 +121,28 @@ public async Task ResetProcessingStatusById(int id)
             {
                 submissionForProcessing.Processing = false;
                 submissionForProcessing.Processed = false;
+                submissionForProcessing.CreatedOn = DateTime.Now.ToUniversalTime();
```

Comment:
This will not be needed if entity is AuditInfoEntity

Reply:
Done